src/ResourceManagement/ApiManagement/ApiManagementManagement/Customizations/IPolicyOperations.cs

```diff
@@ -0,0 +1,26 @@
+﻿//  
+// Copyright (c) Microsoft.  All rights reserved.
+// 
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//    http://www.apache.org/licenses/LICENSE-2.0
+// 
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+namespace Microsoft.Azure.Management.ApiManagement
+{
+    public partial interface IPolicyOperations
```

Comment:
Please add comments over public classes, interfaces that are not part of generated code.